cbt.go

```diff
@@ -1422,11 +1423,20 @@ func doRead(ctx context.Context, args ...string) {
 		log.Fatal(`"start"/"end" may not be mixed with "prefix"`)
 	}
 
+	reversed := false
+	if reversedStr := parsed["reversed"]; reversedStr != "" {
+		reversed, err = strconv.ParseBool(reversedStr)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+
 	var rr bigtable.RowRange
-	if start, end := parsed["start"], parsed["end"]; end != "" {
-		rr = bigtable.NewRange(start, end)
-	} else if start != "" {
-		rr = bigtable.InfiniteRange(start)
+	start, end := parsed["start"], parsed["end"]
+	if reversed {
+		rr = bigtable.NewOpenClosedRange(start, end)
+	} else {
+		rr = bigtable.NewClosedOpenRange(start, end)
 	}
 	if prefix := parsed["prefix"]; prefix != "" {
 		rr = bigtable.PrefixRange(prefix)
```

Comment:
considering the Open-Closed/Closed-Open changes above, should the prefix have a similar behavioral change depending on the "reversed" flag?

Reply:
n/a